storage/tianmu/vc/subselect_column.cpp

```diff
@@ -444,14 +444,15 @@ bool SubSelectColumn::IsNullImpl(const core::MIIterator &mit) {
   return tmp_tab_subq_ptr_->IsNull(0, col_idx_);
 }
 
-types::RCValueObject SubSelectColumn::GetValueImpl(const core::MIIterator &mit, [[maybe_unused]] bool lookup_to_num) {
+types::TianmuValueObject SubSelectColumn::GetValueImpl(const core::MIIterator &mit,
+                                                       [[maybe_unused]] bool lookup_to_num) {
   PrepareSubqResult(mit, false);
-  types::RCValueObject val = tmp_tab_subq_ptr_->GetValueObject(0, col_idx_);
+  types::TianmuValueObject val = tmp_tab_subq_ptr_->GetValueObject(0, col_idx_);
   if (expected_type_.IsString())
     return val.ToBString();
   if (expected_type_.IsNumeric() && core::ATI::IsStringType(val.Type())) {
-    types::RCNum rc;
-    types::RCNum::Parse(*static_cast<types::BString *>(val.Get()), rc, expected_type_.GetTypeName());
+    types::TianmuNum rc;
+    types::TianmuNum::Parse(*static_cast<types::BString *>(val.Get()), rc, expected_type_.GetTypeName());
     val = rc;
```

Comment:
typename is changed from RCNum to TianmNum, the variable name rc is better to changed to tianmu or similar name.

Reply:
Yeah, I agree!